src/runnerversion

```diff
@@ -1 +1 @@
-2.287.0
+2.287.1
```

Comment:
This is here to run locally without the runner trying to download a newer version.

I'm assuming we'll have to revert this change in this PR since we don't want to push a new version once this is merged, correct?

Reply:
This should be reverted, I have a pending change that will address this, its due to a hotfix we had to release off of the releases branch that hasn't been merged into `main` yet